tutorial/index.jade

```diff
@@ -112,9 +113,9 @@
       the first agent, so he can join the same conference.
 
       When providing instructions to the client, we also provide a
-      [`waitUrl`](//www.twilio.com/docs/api/twiml/conference#attributes-waitUrl),
-      which in our case is another end point of our application. This returns more TwiML
-      to [say](//www.twilio.com/docs/api/twiml/say) welcome to the user, and also
+      [`waitUrl`](//www.twilio.com/docs/api/twiml/conference#attributes-waitUrl).
+      This URL is another end point of our application. This returns more TwiML
```

Comment:
What do you think about using _endpoint_ instead of _end point_? Twilio's documentation is using it that way.

1. https://www.twilio.com/docs/api/twilio-sip/receiving-sip-how-it-works
1. https://www.twilio.com/docs/api/rest/test-credentials